src/api/util/utility/passwordStrength.ts

```diff
@@ -41,25 +41,25 @@ export function checkPassword(password: string): number {
 	let pwStrength = 0;
 
 	// checks for total password len
-	if (password.length >= 8 - 1) {
+	if (password.length >= 7) {
```

Comment:
I think these might been hard coded values for min and max that should have been configurable

Reply:
should I add the configuration options back then?